tools/pipeline-witness/Azure.Sdk.Tools.PipelineWitness.Tests/Azure.Sdk.Tools.PipelineWitness.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
Any reason you need net7.0? We usually stick to LTS unless there is a good reason. net8.0 will become the next LTS before long

Reply:
I guess net8.0 is LTS now and 6 will be going out of support in Nov. https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core

We probably should start moving all our tools to net8.0 soon.

Reply:
I've updated to net8.   I needed to change global.json too